### PR TITLE
Design change related to issue #100

### PR DESCRIPTION
**Pull request by nicholsk18**
I had to change the way data is being passed to the template for a cleaner and easier way to loop over that data.
Let me know if something needs to be refactored in `inventoryitem_group.py` file. Python is not my go-to language 😄

### CODE REVIEW

File: netbox_inventory/views/inventoryitem_group.py [hunk 1]
```diff
@@ -49,6 +49,40 @@ def get_extra_context(self, request, instance):
...
+                # make sure skip first insert
+                if len(asset_obj.keys()) != 0:
+                    type_status_objects.append(asset_obj)
+                asset_obj = {}
+
+            # collection of statuses for same types
+            status_list = { 
+                'status': tsc['status'],
+                'count': str(tsc['count']), # needs to be a string to render
+                'color': tsc['color']
```
Comment (matejv) [inline netbox_inventory/views/inventoryitem_group.py:70]:
```suggestion
                'color': tsc['color'],
                'label': tsc['label']
```

Add status label (aka display)

File: netbox_inventory/templates/netbox_inventory/inventoryitemgroup.html [hunk 2]
```diff
@@ -79,33 +79,27 @@ <h5 class="card-header">Asset count by type & status</h5>
...
-                {% badge value=tsc.label bg_color=tsc.color %}
-              </td>
-              <td>
-                <a href="{% url 'plugins:netbox_inventory:asset_list' %}?inventoryitem_type_id={{ tsc.inventoryitem_type }}&status={{ tsc.status }}">
-                  {{ tsc.count }}
-                </a>
+              <td class="d-flex">
+                {% for status in tsc.status_list %}
+                  <a href="{% url 'plugins:netbox_inventory:asset_list' %}?inventoryitem_type_id={{ tsc.inventoryitem_type }}&status={{ status.status }}" class="w-100 me-2">
+                    {% badge value=status.status|capfirst|add:' - '|add:status.count bg_color=status.color|add:' w-100' %}
```
Comment (matejv) [inline netbox_inventory/templates/netbox_inventory/inventoryitemgroup.html:96]:
```suggestion
                    {% badge value=status.label|add:' - '|add:status.count bg_color=status.color|add:' w-100' %}
```

I suggest using status label here. 

At least we use localized custom status values, so displaying them consistently throughout UI is desirable.

**Review by matejv — CHANGES_REQUESTED** [review]
Thank you for your contribution. I made a couple of suggestions to use status label in template.

Other than that the code seems fine and the end result looks great!

**Comment by nicholsk18** [comment]
Thank you for the review, label does make more sense there. I also added overflow, for people who might have many badges, so it doesn't break their UI.

**Pull request merged**